crates/rust-analyzer/Cargo.toml

```diff
@@ -17,6 +17,7 @@ path = "src/bin/main.rs"
 [dependencies]
 anyhow = "1.0.26"
 crossbeam-channel = "0.5.0"
+dissimilar = "1.0.2"
```

Comment:
If you could send a patch for https://github.com/rust-analyzer/expect-test to use dissimilar as well, that'd be sweet :)

Reply:
Will do